src/util/misc.ts

```diff
@@ -54,17 +61,17 @@ export const formatTimestampAsDate = (ts: number) => {
 export const formatTimestampRelative = (ts: number) => {
   let unit
   let delta = now() - ts
-  if (delta < seconds(1, "minute")) {
+  if (delta < MINUTE) {
     unit = "second"
-  } else if (delta < seconds(1, "hour")) {
+  } else if (delta < HOUR) {
     unit = "minute"
-    delta = Math.round(delta / seconds(1, "minute"))
-  } else if (delta < seconds(2, "day")) {
+    delta = Math.round(delta / MINUTE)
+  } else if (delta < 2 * DAY) {
```

Comment:
This is fine, but for future reference I prefer to use `int(2, DAY)` which just makes things a little clearer